lib/tests/streamlit/runtime/memory_session_storage_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import MagicMock
+
+from streamlit.runtime.memory_session_storage import MemorySessionStorage
+
+
+class MemorySessionStorageTest(unittest.TestCase):
+    """Test MemorySessionStorage.
+
+    These tests are intentionally extremely simple to ensure that we don't just end up
+    testing cachetools.TTLCache. We try to just verify that we've wrapped TTLCache
+    correctly, and in particular we avoid testing cache expiry functionality.
+    """
```

Comment:
This is maybe a bit pedantic, but because we're not testing any of TTLCache's functionality directly, I think it's worth having a test that simply asserts that `isinstance(store._cache, TTLCache)`, along with a comment that if a developer breaks this test (by changing the underlying storage type), they're on the hook for testing cache expiry.

Reply:
Seems like a reasonable thing to add a test for